src/main/java/network/brightspots/rcv/SecurityConfig.java

```diff
@@ -22,7 +22,12 @@ class SecurityConfig {
   // Only the unit test modules should ever set this to false, if it is initially set as true.
   // Note: On some builds, this will be configured to false by default. We will need some
   // formalized method of toggling this for two versions of builds, which has yet to be determined.
-  private static boolean IS_HART_SIGNATURE_VALIDATION_ENABLED = true;
+  private static boolean IS_HART_SIGNATURE_VALIDATION_ENABLED = false;
+
+  // Is the user allowed to save output files to their Users directory?
+  // Since user accounts retain delete and create permissions to their user account folders,
+  // this should be disallowed to truly ensure output files are read-only.
+  private static boolean CAN_OUTPUT_FILES_SAVE_TO_USERS_DIRECTORY = true;
```

Comment:
> After approval, I will toggle both security flags and turn them off before merging into develop
> Then, I will cherry-pick into 1.3.2 and toggle the security flags back to what they are here

Does this mean we're going to have to remember to do this every single time we do a launch? It'd be pretty annoying to keep this `false` for development cycles, but it also seems easy to forget about doing this whenever we do a prod release.

Reply:
Yes, and we need a better way to manage these flags. For now, it's nice that they're centrally located so we can turn on strict mode. Eventually, we absolutely need a better way to approach this than hoping to remember to change flags.

Reply:
What if we made a helper function somewhere that checks if the current version is *999* (dev) and, if so, it flips them accordingly. Otherwise, it flips them the other way?

We could use this same function to test whether or not to render the developer version banner in the GUI too.

Reply:
If we do go this route, perhaps we could create the helper in #719 and then merge that in, then make use of it in this PR?

Reply:
I'm looking forward to #719 because I think it's going to make sure we have solid version control. But in the interest of getting a v1.3.2 release candidate out can we defer that work until later? I've run past a couple of my estimates for this build release and the pressure is building to get it out

Reply:
It would be trivial to do this, but from what @artoonie said in Slack, these flags aren't something we'll necessary want flipped the same way for 1.3.2 as in future releases, so I guess it's best to handle them ad hoc now.